docs/root/version_history/v1.10.0.rst

```diff
@@ -5,97 +5,97 @@ Changes
-* access log: added a new field for upstream transport failure reason in :ref:`file access logger<config_access_log_format_upstream_transport_failure_reason>` and
-  :ref:`gRPC access logger<envoy_api_field_data.accesslog.v2.AccessLogCommon.upstream_transport_failure_reason>` for HTTP access logs.
+* access log: added a new field for upstream transport failure reason in :ref:`file access logger <v1.10.0:config_access_log_format_upstream_transport_failure_reason>` and
+  :ref:`gRPC access logger <v1.10.0:envoy_api_field_data.accesslog.v2.AccessLogCommon.upstream_transport_failure_reason>` for HTTP access logs.
 * access log: added new fields for downstream x509 information (URI sans and subject) to file and gRPC access logger.
 * admin: the admin server can now be accessed via HTTP/2 (prior knowledge).
 * admin: changed HTTP response status code from 400 to 405 when attempting to GET a POST-only route (such as /quitquitquit).
 * buffer: fix vulnerabilities when allocation fails.
 * build: releases are built with GCC-7 and linked with LLD.
-* build: dev docker images :ref:`have been split <install_binaries>` from tagged images for easier
+* build: dev docker images :ref:`have been split <v1.10.0:install_binaries>` from tagged images for easier
   discoverability in Docker Hub. Additionally, we now build images for point releases.
 * config: added support of using google.protobuf.Any in opaque configs for extensions.
 * config: logging warnings when deprecated fields are in use.
 * config: removed deprecated --v2-config-only from command line config.
-* config: removed deprecated_v1 sds_config from :ref:`Bootstrap config <config_overview_bootstrap>`.
-* config: removed the deprecated_v1 config option from :ref:`ring hash <envoy_api_msg_Cluster.RingHashLbConfig>`.
-* config: removed REST_LEGACY as a valid :ref:`ApiType <envoy_api_field_core.ApiConfigSource.api_type>`.
+* config: removed deprecated_v1 sds_config from :ref:`Bootstrap config <v1.10.0:config_overview_v2_bootstrap>`.
+* config: removed the deprecated_v1 config option from :ref:`ring hash <v1.10.0:envoy_api_msg_Cluster.RingHashLbConfig>`.
+* config: removed REST_LEGACY as a valid :ref:`ApiType <v1.10.0:envoy_api_field_core.ApiConfigSource.api_type>`.
 * config: finish cluster warming only when a named response i.e. ClusterLoadAssignment associated to the cluster being warmed comes in the EDS response. This is a behavioural change from the current implementation where warming of cluster completes on missing load assignments also.
 * config: use Envoy cpuset size to set the default number or worker threads if :option:`--cpuset-threads` is enabled.
-* config: added support for :ref:`initial_fetch_timeout <envoy_api_field_core.ConfigSource.initial_fetch_timeout>`. The timeout is disabled by default.
-* cors: added :ref:`filter_enabled & shadow_enabled RuntimeFractionalPercent flags <cors-runtime>` to filter.
-* csrf: added :ref:`CSRF filter <config_http_filters_csrf>`.
```

Comment:
there is no documented link anchors for csrf in v1.10.0 (and not really any docs - although there is a very vague mention)